cpp/ql/src/Likely Bugs/Likely Typos/FutileParams.qhelp

```diff
@@ -5,18 +5,25 @@
 
 
 <overview>
-<p>A function is called with arguments despite having an empty parameter list. This may indicate
-that the incorrect function is being called, or that the author misunderstood the function.</p>
+<p>A function is called with arguments despite having been <i>declared</i> with an empty parameter list.
```

Comment:
The query is now broader than this sentence suggests. For example, `declared_empty_defined_with(); // BAD` is not a call with arguments.

Also, the `@name`, `@description`, and alert message are out of date.